code/game/machinery/washing_machine.dm

```diff
@@ -268,6 +268,13 @@ GLOBAL_LIST_INIT(dye_registry, list(
 		slowdown = SHOES_SLOWDOWN
 		new /obj/item/restraints/handcuffs(loc)
 	..()
+	
+/obj/item/clothing/shoes/clown_shoes/machine_wash(obj/machinery/washing_machine/WM)
+	if(slowdown == 15) //I assure you this hack check works.
```

Comment:
```suggestion
	if(istype(src, /obj/item/clothing/shoes/clown_shoes/prison))
```
okay, THIS time, it should work

I had forgotten to add an "src" in there the first time